tap_mixpanel/__init__.py

```diff
@@ -17,7 +17,6 @@
 REQUEST_TIMEOUT = 300
 REQUIRED_CONFIG_KEYS = [
     "project_timezone",
-    "api_secret",
```

Comment:
What's the reason for removing this?

Reply:
We have provided support of `service account authentication` and if any new user want to configure tap with this authentication approach then user do not require to provide `api_secret`. Instead, user will require to provide following parameter values,

```
service_account_username
service_account_secret
project_id
```